src/test/java/core/operationstrategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.operationstrategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.service.OperationType;
+import core.transactions.BalanceOperationHandler;
+import core.transactions.OperationHandler;
+import core.transactions.PurchaseOperationHandler;
+import core.transactions.ReturnOperationHandler;
+import core.transactions.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyImplTest {
+    @Test
+    public void testGetOperationHandler_ok() {
+        OperationHandler balanceHandler = new BalanceOperationHandler();
```

Comment:
Please make a separate test for each operation, also add all commons to beforeAll